.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 test-report.*
 *.xcresult/
+Package.resolved
```

Comment:
Thank you for this Levi, I have been reading up a lot on the SPM lock file and its benefits and its advantages outweigh the annoyance. 

- Even though we allow a range of internal dependencies and imagineing there was a faulty version in it. It can help user to reproduce and diagnose any issues that arise due to changes in the dependencies. Because the lock file specifies the exact versions of your dependencies that were used when building your SDK. 
- Secondly, the lock file can help to ensure that all collaborators are using the same versions of dependencies when building and testing the SDK.

Reply:
I am okay with trying this.